docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -693,6 +693,6 @@ The response returns the entry that has just been deleted.
 
 ## Configuration in production environments
 
-A `STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE` [environment variable](/developer-docs/latest/setup-deployment-guides/configurations.md#environment-variables) can be configured to set the default locale for your environment.
+A `STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE` [environment variable](/developer-docs/latest/setup-deployment-guides/configurations.md#environment-variables) can be configured to set the initialization locale for your environment. The value used for this variable should be a string (see [full list of available locales](https://github.com/strapi/strapi/blob/features/i18n/packages/strapi-plugin-i18n/constants/iso-locales.json)).
 
-The value used for this variable should be a string (see [full list of available locales](https://github.com/strapi/strapi/blob/features/i18n/packages/strapi-plugin-i18n/constants/iso-locales.json)).
+This is useful when a Strapi app is deployed in production, with the i18n plugin installed and enabled on your content types for the first time. On a fresh i18n plugin installation, `'en'` is the default locale. So if the database does not contain any locale, and no `STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE` is set for the environment, all the existing content saved in the database will be migrated with the locale defined as `'en'`. But if the `STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE` is defined, then the content will be migrated and set up with this locale. This will prevent from manually updating the locale for every existing content entry.
```

Comment:
instead of all the existing content I would have said that the content of the content types with i18n enabled will be auto migrated to the locale. The rest looks good to me!

Reply:
Thanks @alexandrebodin ! I've just [updated it](https://github.com/strapi/documentation/pull/224/commits/8001b388c89a2ef55c0166656daf16bc888c1f9b) :-)